### PR TITLE
Add smoke tests + save pickled model for KMeans

### PR DESCRIPTION
**Pull request by twolodzko**
* We didn't have any tests for this library, so I added a basic smoke test (log to Neptune + check the basic structure, w/o any detailed checks).
* When doing the above, I found that we are not saving the pickled model for KMeans. This was trivial to add and I did so.

### CODE REVIEW

**Comment by AleksanderWWW** [comment]
Maybe we could also add running tests to the github workflow?

**Comment by twolodzko** [comment]
@AleksanderWWW good idea, done.

File: src/neptune_sklearn/impl/__init__.py [hunk 1]
```diff
@@ -369,7 +372,9 @@ def get_pickled_model(estimator):
             run = neptune.init(project='my_workspace/my_project')
             run['estimator/pickled_model'] = npt_utils.get_pickled_model(rfr)
     """
-    assert is_regressor(estimator) or is_classifier(estimator), "Estimator should be sklearn regressor or classifier."
+    assert (
+        is_regressor(estimator) or is_classifier(estimator) or isinstance(estimator, KMeans)
+    ), "Estimator should be sklearn regressor or classifier."
```
Comment (SiddhantSadangi) [inline src/neptune_sklearn/impl/__init__.py]:
```suggestion
    ), "Estimator should be sklearn regressor, classifier, or Kmeans instance."
```

**Review by kshitij12345 — CHANGES_REQUESTED** [review]
I think for dicts it would make sense to wrap them in `stringify_unsupported`.

Also, a lot of docstrings have `neptune.init()` so we may want to update that.

**Comment by twolodzko** [comment]
@kshitij12345 this PR is not about compatibility with v 1.0.0 but about adding tests. Compatibility issues are fixed in the other ones.

**Comment by kshitij12345** [comment]
Noted!

**Comment by twolodzko** [comment]
@kshitij12345 then can you accept it, or is there anything to change?

**Review by kshitij12345 — APPROVED** [review]
LGTM (on code-front). I am not confident about the CI workflow so would be better that someone else goes through them. (cc: @SiddhantSadangi @AleksanderWWW )

Thank you @twolodzko

**Review by AleksanderWWW — APPROVED** [review]

**Pull request merged**